tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -752,7 +752,19 @@ function updateLocalJestConfig(tree: Tree, options: NormalizedSchema) {
 
   tree.write(options.paths.jestConfig, templates.jest(config));
 
-  if (!tree.exists(jestSetupFilePath)) {
+  if (tree.exists(jestSetupFilePath)) {
+    const content = tree.read(jestSetupFilePath)!.toString('utf-8');
+    const newContent = content.replace(
```

Comment:
if you revert those tests I mentioned (or at least part of it), this will be properly covered

Reply:
so because now we resolved the a11y package, enzyme wont be used indeed.

- this replacement can be removed 
- basically change in this file should be reverted

Reply:
Reverted this file